theme/sass/components/_card-component.scss

```diff
@@ -40,6 +44,14 @@
       padding-left: 15px;
       display: block;
 
+      &.hidden {
+        display: none;
+      }
+
+      &.visible {
+        display: block;
+      }
+
       &:not(:last-child) {
         margin-bottom: 2rem;
       }
```

Comment:
This is preventing there being spacing between the last card and the show less button when additional content is being fixed, please fix this.

Reply:
done